client/App.tsx

```diff
@@ -62,45 +23,24 @@ export default function HomePage() {
                         />
                     </div>
                     <div id="pageOneTextBox">
-                        <h1>GraphQL Rate Limiter</h1>
-                        <h2>GraphQL Gateway developer portal</h2>
-
+                        <h1>GraphQLGate</h1>
+                        <h2>
+                            An Open Source GraphQL rate-limiter with query complexity analysis for
+                            Node.js and Express
+                        </h2>
                         <p>
-                            GraphQL Gateway developer portal is designed for offering a
-                            visualization of how your GraphQL API endpoints is secured using rate
-                            limits and depth limits. With this visulaization tool, users can:
+                            Estimate the upper bound of the response size before resolving the query
+                            and optionally log query data to our Gateway Developer Portal to monitor
+                            and tune rate limiting settings.
                         </p>
-                        <span>
-                            <li>
-                                Visualize API call data and facilitate a tuning of rate limiting
-                                algorithm settings
-                            </li>
-                            <li> Seek for query resolution optimizations for GraphQL APIs </li>
-                            <li>View cached performance metrics</li>
-                        </span>
                     </div>
                 </section>
                 <section id="demoCard">
                     <div id="demoCardWrapper">
-                        <div className="algorithmCard">
-                            <h3>Alghorhitm</h3>
-                            <h1>Token Bucket</h1>
-                            <img src={gear} alt="settings" />
-                        </div>
-                        <div className="algorithmCard">
-                            <h3>Alghorhitm</h3>
-                            <h1>Sliding Window</h1>
-                            <img src={gear} alt="settings" />
-                        </div>
-                        <div className="algorithmCard">
-                            <h3>Alghorhitm</h3>
-                            <h1>Fixed Window</h1>
-                            <img src={gear} alt="settings" />
-                        </div>
-                        <div className="demoCardImg" />
+                        <img src={snippet} alt="settings" />
                     </div>
                 </section>
-                <section id="pageTwo" />
+                <section style={{ height: '555px' }} id="pageTwo" />
                 <div className="centeredSentence">
                     <h1>What makes us different?</h1>
                 </div>
```

Comment:
This is where the updated content would go: still showing `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed doeiusmod tempor incididunt ut labore et dolore magna aliqua.` which looks hack.